mock/src/test/java/feign/mock/MockClientSequentialTest.java

```diff
@@ -20,45 +20,48 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.Matchers.startsWith;
 import static org.junit.Assert.fail;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.lang.reflect.Type;
-import java.util.List;
-import javax.net.ssl.HttpsURLConnection;
-import org.junit.Before;
-import org.junit.Test;
+
 import feign.Body;
 import feign.Feign;
 import feign.FeignException;
+import feign.Headers;
 import feign.Param;
 import feign.RequestLine;
 import feign.Response;
 import feign.codec.DecodeException;
 import feign.codec.Decoder;
 import feign.gson.GsonDecoder;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Type;
+import java.util.List;
+import javax.net.ssl.HttpsURLConnection;
+import org.junit.Before;
+import org.junit.Test;
 
 public class MockClientSequentialTest {
 
   interface GitHub {
 
+    @Headers({"Name: {owner}"})
     @RequestLine("GET /repos/{owner}/{repo}/contributors")
     List<Contributor> contributors(@Param("owner") String owner, @Param("repo") String repo);
 
     @RequestLine("GET /repos/{owner}/{repo}/contributors?client_id={client_id}")
     List<Contributor> contributors(@Param("client_id") String clientId,
-                                   @Param("owner") String owner,
-                                   @Param("repo") String repo);
+      @Param("owner") String owner,
+      @Param("repo") String repo);
 
     @RequestLine("PATCH /repos/{owner}/{repo}/contributors")
     List<Contributor> patchContributors(@Param("owner") String owner, @Param("repo") String repo);
 
     @RequestLine("POST /repos/{owner}/{repo}/contributors")
     @Body("%7B\"login\":\"{login}\",\"type\":\"{type}\"%7D")
     Contributor create(@Param("owner") String owner,
-                       @Param("repo") String repo,
```

Comment:
Looks like you changed the formatting for many classes.... could you please rollback?

running `mvn clean install` should do it.

Reply:
Yep, of course. I'll plug the eclipse formatter plugin to my IDE.